### ISSUE TITLE
Error: File (centos-7.ova.vmdk) could not be found

### ISSUE THREAD

**Issue by luniHw** [kubernetes-sigs/cluster-api-provider-vsphere#896]
/kind bug

**What steps did you take and what happened:**
root@bootstrapper [ ~ ]# ovftool --network="VM Network" --acceptAllEulas --noSSLVerify --allowExtraConfig --X:injectOvfEnv --datastore="sharedVmfs-0" --name="istrator@vsphere.local':'Admin!23'@192.168.111.5/'os-test-dc'/host/compute_clustercentos-7-kube-v1.17.3.ova' vi://'admini
Opening OVA source: http://storage.googleapis.com/capv-images/release/v1.17.3/centos-7-kube-v1.17.3.ova
The manifest validates
Opening VI target: vi://administrator%40vsphere.local@192.168.111.5:443/os-test-dc/host/compute_cluster
Deploying to VI: vi://administrator%40vsphere.local@192.168.111.5:443/os-test-dc/host/compute_cluster
Transfer Failed
Error: File (centos-7.ova.vmdk) could not be found.
Completed with errors

**What did you expect to happen:**

**Anything else you would like to add:**
I use ovftool upload http://storage.googleapis.com/capv-images/release/v1.16.7/centos-7-kube-v1.16.7.ova, it works. Fail to upload centos-7-kube-v1.17.3.ova

**Environment:**

- Cluster-api-provider-vsphere version: 
- Kubernetes version: (use `kubectl version`): 
- OS (e.g. from `/etc/os-release`):

**Comment by codenrhoden** [comment]
Thanks for the report @luniHw and the ping @yastij. I'll try to look into this in the next day or two! Not sure what could have gone wrong between the image that works and the one that doesn't, but I'll find it!

**Comment by codenrhoden** [comment]
I download this OVA and peeked inside. It definitely has the VMDK in it:

```sh
$ tar tf centos-7-kube-v1.17.3.ova
centos-7.ovf
centos-7.mf
centos-7.ova.vmdk
```

I also verified that both the `.mf` file and the `.ovf` file referenced the right file.

I also imported this OVA into Fusion, and it imported correctly. So unless there was something specific about import to vCenter to cause this, I'm inclined to think there was a download error.

I have also recently uploaded a new OVA for `1.17.5` if you want to give that a try. But right now, I am unable to recreate this.
